Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation3D.cxx

```diff
@@ -398,6 +401,9 @@ void vtkSlicerMarkupsWidgetRepresentation3D::CanInteract(
       }
     }
 
+  // See if we can interact with the handle
```

Comment:
What about describing what is done to check if interaction can effectively be done or not.